cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -837,64 +852,49 @@ instance ( tx ~ GenTx blk
          , ToObject (CannotForge blk)
          , ToObject (ForgeStateUpdateError blk))
       => ToObject (TraceForgeEvent blk) where
-  toObject MaximalVerbosity (TraceAdoptedBlock slotNo blk txs) =
+  toObject _verb (TraceStartLeadershipCheck slotNo) =
     mkObject
-      [ "kind" .= String "TraceAdoptedBlock"
```

Comment:
"TraceAdoptedBlock" is an important event for metrics in benchmarking. The reordering in this file makes it difficult to assess if this is still traced.

Reply:
I did this for a good reason, to stick to the same order as in which they're defined: https://github.com/input-output-hk/ouroboros-network/blob/7cda58405b6ee0d335b11e88e5c9989c7a3a6e03/ouroboros-consensus/src/Ouroboros/Consensus/Node/Tracers.hs#L143-L310

IMO, by maintaining the same order, it is easier to spot (by eye) which events are already handled and which are missing (the compiler tells you this as well, of course).